source/opt/dominator_tree.cpp

```diff
@@ -242,6 +242,7 @@ bool DominatorTree::Dominates(uint32_t a, uint32_t b) const {
 bool DominatorTree::Dominates(const DominatorTreeNode* a,
                               const DominatorTreeNode* b) const {
   // Node A dominates node B if they are the same.
+  if (!a || !b) return false;
```

Comment:
Can you move the comment at line 244 just before the condition that it is describing?